src/database/db_build.sql

```diff
@@ -0,0 +1,46 @@
+BEGIN;
+
+DROP TABLE IF EXISTS mentors, questions, singing cascade;
+
+CREATE TABLE mentors(
+  id        SERIAL    PRIMARY KEY,
+  name      TEXT      NOT NULL,
+  imageUrl  TEXT      NOT NULL
+);
+
+CREATE TABLE questions(
+  id          SERIAL    PRIMARY KEY,
+  category    TEXT      NOT NULL,
+  question    TEXT      NOT NULL
+);
+
+CREATE TABLE singing(
+  id        SERIAL        PRIMARY KEY,
+  name      VARCHAR(20)   NOT NULL,
```

Comment:
:+1: good spot I think we could call the column mentor_id and have it equal the id in the mentors table. Thoughts?